src/main/cpp/t04_swap.cpp

```diff
@@ -25,6 +25,28 @@
 using namespace std;
 
 int t04_swap() {
+    int n, m;
+    cin >> n >> m;
+    int a[100][100];
+
     
+    for (int k = 0; k < n; k++) {
+        for (int b = 0; b < m; b++) {
+            cin >> a[k][b];
+        }
+    }
+    int i,j, c=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'c' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4719290)